tests/conftest.py

```diff
@@ -67,12 +68,13 @@ def create_good_xml(tmp_path_factory):
     with open(os.path.join(str(input_dir), 'temp.xml'), 'w', encoding='utf-8') as temp_xml:
         temp_xml.write(xmltodict.unparse(doc))
 
-    return str(input_dir)
+    return input_dir, output_dir
```

Comment:
nice, i noticed this problem in a different PR.  Thought to make the change in a different branch and submit PR from there, but here is good too!